sonar-plugin-api/src/main/java/org/sonar/api/server/rule/DefaultRuleDescriptionSection.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * SonarQube
```

Comment:
Is the header title still SonarQube in this new `sonar-plugin-api` project ?

Reply:
the HEADER file needs to be changed - out of scope of this PR.
I'll create a ticket